tutorials/get_started/install.py

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Installing TVM
+==============
+**Authors**:
+`Jocelyn Shiue <https://github.com/>`_,
+`Chris Hoge <https://github.com/hogepodge>`_
+
+Depending on your needs and your working environment, there are a few different
+methods for installing TVM. These include:
+    * Installing from the TLCPack Conda and Pip Packages
+    * Installing from source
+"""
+
+################################################################################
+# Installing from TLC Pack
+# ------------------------
+# TVM is packaged and distributed as part of the volunteer TLCPack community.
+# TLCPack is not affiliated with the Apache Software Foundation, and software
+# distributed by them may include non-free hardware drivers. You can lean more
+# about TLCPack at their `website <https://tlcpack.ai>`_. TLCPack offers a
+# support matrix with instructions for how to install TVM on different
+# platforms, with different features. If installing with pip or conda through
+# TLCPack, we recommend to use one of the nightly builds, as they contain the
+# latest version of `tvmc`. Some platforms and configurations are not
+# available.  If your environment isn't supported, you can install TVM from
+# source.
+
+################################################################################
+# Installing from Source
```

Comment:
move source to the beginning as officially we release only the source pkg

Reply:
done